backend/test/card_game/test_helper.clj

```diff
@@ -0,0 +1,20 @@
+(ns card-game.test-helper
+  (:require [card-game.core :as core]
+            [configs :as configs]))
+
+(defn end-game
+  ([game-state] (end-game game-state 0))
```

Comment:
if we base the exit condition on the current number of cards in hand extracted from the game-state, we don't need the previous-plays parameter.

Reply:
I don't want to use the game-state because we are (sometimes) testing that the game-state correctly tracks the number of cards in hand.

Reply:
But that shouldn't be what this function does, right? I think the purpose of the function is to drive the game to an end no matter what. And I don't recall seeing any test where we use this function with an incorrect number of previous plays (we might, I just don't recall it)

Reply:
Will do that. Maybe that also helps adding strategies